torch/csrc/jit/codegen/fuser/tensor_info.h

```diff
@@ -18,12 +17,8 @@ struct TORCH_API TensorInfo {
   }
 
   void* data;
-#pragma GCC diagnostic ignored "-Wpedantic"
```

Comment:
Hmm, we sure we don't need this anymore? I am not sure if we have pedantic enabled generally throughout the codebase so I'd leave it in.

Reply:
Enabled globally in some previous PR.